js/script.js

```diff
@@ -1,61 +1,122 @@
 'use strict';
 
-var output = document.getElementById('output');
+var welcomeMessage = document.getElementById('welcomeMessage');
+var showResults = document.getElementById('showResults');
 var showNumberOfRounds = document.getElementById('showNumberOfRounds');
-
-output.innerHTML = 'Witamy w grze kamień papier nożyce';
-
-function askHowManyRounds() {
-        console.log(1);
-        var numbersOfRounds = prompt('Ile rund?');
-        console.log(2);
-        return numbersOfRounds;
-        console.log(3);
-        
-        console.log(4);
-        
+var showWhoWon = document.getElementById('showWhoWon');
+var endOfGame = document.getElementById('endOfGame');
+var beginButton = document.getElementById('beginButton');
+var winner = document.getElementById('wins');
+var remixed = document.getElementById('draws');
+var losser = document.getElementById('loss');
+var games = document.getElementById('games');
+var totalRoundsForGame;
+var wins = 0;
+var loss = 0;
+var game = 0;
+var draws = 0;
+winner.innerHTML = wins;
+remixed.innerHTML = draws;
+losser.innerHTML = loss;
+games.innerHTML = game;
+
+
+startGame();
+
+function startGame() {
+    console.log('START GAME')
+    welcomeMessage.innerHTML = 'Witamy w grze kamień papier nożyce';
+    registerListeners();
 };
-var numberOfRounds = askHowManyRounds();
-showNumberOfRounds.innerHTML = 'ilość rund: ' + askHowManyRounds();
-
-
-beginButton.addEventListener('click', function() {
-        
-       document.getElementById('stone').style.display = "block";
-       document.getElementById('paper').style.display = "block";
-       document.getElementById('scissors').style.display = "block";
-       askHowManyRounds();
-});     
-
-
 
+function askHowManyRounds() {    
+    var numbersOfRounds = prompt('Ile rund?');  
+    if (isNaN(numbersOfRounds) || numbersOfRounds <= 0) {
+            alert('Podałeś nieprawidłową wartość licznik ustawiony na 5 rund');
+            return numbersOfRounds = 5;       
```

Comment:
1) wystarczy samo `return '5'` - wiesz dlaczego?
2) To powinien być string 5 nie liczba - wiesz dlaczego?